README.md

```diff
@@ -96,12 +96,12 @@ If Mapperly cannot generate a mapping, one can implement it manually simply by p
 
 ```c#
 [Mapper]
-public interface IDtoMapper
+public partial class DtoMapper
 {
-    CarDto CarToCarDto(Car car);
+    partial CarDto CarToCarDto(Car car);
 
     DateOnly DateTimeToDateOnly(DateTime dt) => DateOnly.FromDateTime(dt);
```

Comment:
I would add the `public` keyword to the mapping method and the `private` keyword to the user implemented method (assuming that works).

Reply:
Fixed.